test/utils/mocks/modules/paymentProcessor/PP_Streaming_v1AccessMock.sol

```diff
@@ -28,30 +28,30 @@ contract PP_Streaming_v1AccessMock is PP_Streaming_v1 {
         return unclaimableAmountsForStream[client][token][sender][id];
     }
 
-    function original_validPaymentReceiver(address addr)
+    function original__validPaymentReceiver(address addr)
```

Comment:
All these double underscores are probably ctrl-r mistake, right? I personally don't mind them, but just wanted to point it out

Reply:
Will update. We were missing underscores with the internal functions, and when replacing them in the code base, the mock functions exposing internal functions got updated as well